py/obj: fix spelling of truncated

dlech opened the pull request.

dpgeorge commented:
Might I suggest instead that this entire TODO comment be removed, and instead recorded as a GitHub issue discussing this possible optimisation?  On the one hand it's nice to have such comments in the code for people to discover and think about, but on the other hand it's not really sustainable to do it for all ideas.  This comment was from very early on in the project.

dlech committed:
py/obj: remove TODO idea comment

It was suggested to move this to a GitHub issue rather than keep it
in the code.

dlech commented:
updated

dpgeorge commented:
Thanks, merged in f020eac6a85fb956dbd61ff6a138ad94ac87a688
